easygems/healpix/__init__.py

```diff
@@ -43,6 +43,14 @@ def isel_extent(dx, extent):
     return np.arange(get_npix(dx))[get_extent_mask(dx, extent)]
 
 
+def broadcast_array(dx, fill_value=np.nan):
+    """Broadcast a limited-area HEALPix array to full shape."""
+    arr = fill_value * np.ones(shape=get_npix(dx))
+    arr[dx.cell] = dx.values
```

Comment:
Maybe you'd want to replace actually both of the lines with something based on either `xarray.DataArray.where`, or the three-argument version of `numpy.where`.

Reply:
I don't see how we use `where` to both broadcast the sparse array **and** fill the corresponding values 🤔 But we can also discuss the general approach offline 😉

Reply:
Probably that's right 🤔